src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceReferenceContact.vue

```diff
@@ -64,72 +64,110 @@
           ></v-text-field>
         </v-col>
       </v-row>
-      <h2 class="mt-5">ECE certification</h2>
-      <div v-if="wizardStore.wizardData.inviteType === PortalInviteType.CHARACTER" role="doc-subtitle">
-        If you are registered as an ECE in Canada, please provide your certification number.
+      <div v-if="wizardStore.wizardData.workExperienceType == WorkExperienceType.IS_400_Hours">
+        <h2 class="mt-5">ECE certification</h2>
+        <div>If you're registered as an ECE in Canada, please provide your certification number.</div>
+
+        <v-row class="mt-5">
+          <v-col cols="12" md="8" lg="6" xl="4">
+            <v-autocomplete
+              :model-value="modelValue.certificateProvinceId"
+              label="Province/Territory Certified/Registered In (Optional)"
+              variant="outlined"
+              color="primary"
+              :items="configStore?.provinceList"
```

Comment:
I think we need to filter out the "other" option from the province list... might be worth creating it in the store as a getter if we have to do this multiple times. 

I noticed I did it for the province lists below.

Reply:
Keeping this aligned to what design asks atm, will adjust later if QA/Design feedback says so